assets/js/googlesitekit-datastore-forms.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Entrypoint for the core/forms data store.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import './googlesitekit/datastore/forms';
```

Comment:
@tofumatt is there anything I should do to mitigate this side-effect ?

Reply:
There's no need to mitigate, I meant it's worth adding a comment so folks know that's what's happening here if they see the code; otherwise it looks like an import that's not doing anything.

Reply:
Extra comment has been added